jbpm-flow-builder/src/main/java/org/jbpm/weaver/BPMN2WeaverService.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.weaver;
+
+import org.kie.api.io.ResourceType;
+
+public class BPMN2WeaverService extends AbstractWeaverService {
```

Comment:
this info is already available in BPMN2AssemblerService (the resource type). Any way to combine and not have two?

Reply:
I agree. I may think of a way, but unfortunately weaver and assembler are supposed to be separate. TBF I think this API should benefit another round of review, but this is what we have available right now. See the related JIRA for my thoughts on that.